snapshots/devmapper/pool_device.go

```diff
@@ -350,19 +368,34 @@ func (p *PoolDevice) DeactivateDevice(ctx context.Context, deviceName string, de
 	return nil
 }
 
-// IsActivated returns true if thin-device is activated and not suspended
+// IsActivated returns true if thin-device is activated
 func (p *PoolDevice) IsActivated(deviceName string) bool {
 	infos, err := dmsetup.Info(deviceName)
 	if err != nil || len(infos) != 1 {
 		// Couldn't query device info, device not active
 		return false
 	}
 
+	if devInfo := infos[0]; devInfo.TableLive {
+		return true
+	}
+
+	return false
+}
+
+// IsSuspended returns true if thin-device is suspended
+func (p *PoolDevice) IsSuspended(deviceName string) bool {
```

Comment:
nit: so `IsSuspended` not actually needed, right?

Reply:
yes, thanks!